client/package.json

```diff
@@ -42,6 +42,12 @@
                     "default": 100,
                     "description": "Controls the maximum number of problems produced by the server."
                 },
+                "yseopml.pathToPredefinedObjectsYml": {
+                    "scope": "window",
+                    "type": "string",
+                    "default": "",
+                    "description": "Full path to an xml file containing the YML types definition."
```

Comment:
“xml” ↔ “YML”
Please be more consistent regarding acronyms. I would advocate using uppercase letters for all “—ML” stuff.
Oh, and as I said somewhere else, use “absolute” or “relative path” rather than “full path”. I never really know what you mean by that.